src/components/VideoControls/VideoControls.js

```diff
@@ -33,18 +32,29 @@ const VideoControls = (props) => {
             video.pause();
         }
     };
+    const stopVideo = () => (video.currentTime = video.duration);
 
     useEffect(() => {
         video.addEventListener('loadedmetadata', () => {
+            const timeLeftOff = localStorage.getItem(`vid--name--${videoName}`);
```

Comment:
we can just declare `const videoStorageKey = 'vid--name--${videoName}'` as a variable at the top of the component, and re-use it everywhere else